coding_systems/snomedct/tests/test_ecl_parser.py

```diff
@@ -24,6 +26,40 @@ def test_handle(subtests):
             assert handled["excluded"] == excluded
 
 
-def test_parse_error():
+def test_parse_error_syntax_error():
     with pytest.raises(ParseError):
         handle("(111111 OR 222222) MINUS (333333 OR 444444")
+
+
+def test_parse_error_attempting_full_context():
+    with pytest.raises(ParseError, match="Attempting full context"):
+        handle("111111 < 222222")
+
+
+def test_parse_error_ambiguity():
```

Comment:
This is the same as the test above.

Reply:
Ah yes, this one should have been deleted; copied and pasted to add a test for the ambiguity error, but I couldn't find a simple example that would raise one.  Afaict, Ambiguity Error is an antlr4 thing, where the expression could be parsed in more than one way, but I couldn't find an expression that would cause an ambiguity without hitting a different error first.